app/views/messages/homepage.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Chitter</h1>
+<% if current_user %>
+	<form action='/message' method='post'>
+	  <label for='message'>Message:</label>
+	    <input type='text' name='peep'>
+	  <input type='submit' value='Post'>
+	</form>
+<% end %>
+<ul id='messages'>
+	<% @messages.each do |message| %>
+		<li>
+			<label id='username'>
+				<%= User.get(message.user_id).username %> 
+			</label>
+			<label id='date_time'>
+				<%= message.created_at.to_s[11..15] %> <%= message.created_at.to_s[4..9
```

Comment:
Too much logic in your view - this much logic should be unit tested and locked away in the model so that you can just call `message.display_timestamp` or something similar.